content/pages/pricing.html

```diff
@@ -2,389 +2,263 @@
+      {{ pricing.feature_row("Automation rules", "Automatically manage versions", "fa-robot") }}
+
+      {{ pricing.header_row("Documentation hosting", "fa-browser") }}
+      {{ pricing.feature_row("Documentation versioning", "Host multiple versions of your project", "fa-layer-group") }}
+      {{ pricing.feature_row("Documentation localization", "Host your project localized in multiple locales", "fa-language") }}
+      {{ pricing.feature_row("Reader authentication", "Require access to view your documentation", "fa-shield-keyhole", on_community=False) }}
+      {{ pricing.feature_row("Single sign-on", "Sign on with using external provider", "fa-right-to-bracket", on_community=None, on_basic="GitHub and GitLab", on_advanced="GitHub and GitLab", on_pro="GitHub, GitLab, and Google") }}
+      {{ pricing.feature_row("Sharing", "Provide temporary access to private documentation", "fa-user-shield", on_community=False) }}
+      {{ pricing.feature_row("CDN hosting", "Reduce load times for your projects", "fa-clouds", on_community=True, on_basic="Public projects only", on_advanced="Public projects only", on_pro="Public projects only") }}
+      {{ pricing.feature_row("Custom domains", "Host projects on custom domain name", "fa-at", on_basic=False) }}
+      {{ pricing.feature_row("Ad supported", "Documentation includes advertising placements", "fa-ad", on_basic=False, on_advanced=False, on_pro=False) }}
+      {{ pricing.feature_row("Subprojects", "Organize projects using project nesting", "fa-diagram-nested") }}
+      {{ pricing.feature_row("Integrated search", "Search across all of your projects", "fa-magnifying-glass") }}
+      {{ pricing.feature_row("Offline formats", "Download offline PDF and EPUB formats", "fa-file-pdf") }}
+      {{ pricing.feature_row("Request redirects", "Redirect requests for moved and deleted content", "fa-diamond-turn-right") }}
+      {{ pricing.feature_row("Sitemap generation", "Automatically update your project's sitemap", "fa-sitemap") }}
+      {{ pricing.feature_row("Customizable error pages", "Manage your own HTTP error pages", "fa-palette") }}
+      {{ pricing.feature_row("Status badges", "Embeddable badges showing your project's status", "fa-badge-check") }}
+      {{ pricing.feature_row("Google analytics integration", "Track visits to your project", "fa-chart-line") }}
+      {{ pricing.feature_row("Pageview analytics", "Collect statistics on visits to your projects", "fa-chart-line", on_community="30 days", on_basic="30 days", on_advanced="30 days", on_pro="90 days") }}
+      {{ pricing.feature_row("Search analytics", "Collect statistics on search queries for your projects", "fa-chart-line", on_community="30 days", on_basic="30 days", on_advanced="30 days", on_pro="90 days") }}
+      {{ pricing.feature_row("Content embedding", "Embed your documentation in outside sources", "fa-object-subtract", on_basic=False) }}
```

Comment:
I do wonder about putting all the features that are differentiated at the top? That way we have a nice visual for what is included on each plan, and then all the joint features below? Not 100% sure this is better, but I find it hard to easily scan what is diff between plans.

Reply:
There wasn't a clear way to organize this list, because even as we get lower on the features table, the community plan still has features that the basic plan doesn't, so it's hard to organize by plan like what you're describing.

I did find it easy to scan by splitting up by concern -- build, hosting, etc -- but this doesn't help highlight any particular plan necessarily.

Reply:
Yea, I think splitting by section is good, I was mostly suggesting the sorting within the section. Definitely not a fully baked thought though.

Reply:
I put this one on #167 as well, but where I stand right now is that major features should be differentiated in the top plan section instead. The feature table should be more organized -- grouped by function and context makes the list easier to scan.